tools/depmat.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+
+import sys
+from getopt import gnu_getopt, GetoptError
+from re import compile as Regex
+from collections import namedtuple
+
+from libgcopdd import Event, UserError, args_to_events, n_args_to_n_events, open_dump_entry, create_temp
+import phasestack
+
+INTEGER_PATTERN = Regex(r'\d+')
+DepMat = namedtuple('DepMat', 'size, mapping, data')
+
+####
+
+def load(event):
```

Comment:
In this case, it might be just me, but I love having type hints written in every method signature. It makes some things so much easier to understand. For example, in this specific case, I am not sure, whether event is a string or something else.